src/book/02-system/01-foundations/05-mathematics.mdx

```diff
@@ -239,7 +239,7 @@ This gives the **foot** to **torso** transform by **multiplying** the **world**
 
 These two **inner** letters are **cancelled**, with the outer two remaining. This also works for rotation matrices.
 
-If you have two **vectors** in the **same space** and their **inner** (or **outer**, since vector addition is **commutative**) **letters** are both the same then they can be **canceled** in a similar way to the transformation matrices.
+If you have two **vectors** in the **same space** and their **inner** (or **outer**, since vector addition is **commutative**) **letters** are both the same then they can be ** cancelled** in a similar way to the transformation matrices.
```

Comment:
hmmm, prettier should have caught this...

```suggestion
If you have two **vectors** in the **same space** and their **inner** (or **outer**, since vector addition is **commutative**) **letters** are both the same then they can be **cancelled** in a similar way to the transformation matrices.
```